README.md

```diff
@@ -80,5 +80,4 @@
 
 - GitHub: [@githubhandle](https://github.com/fitz95)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] **Your project is deployed 👍 👍** however, some of the necessary headers are missing in your Readme 😨
    - I think it would be better to provide a **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 
    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

    - Kindly make sure that all main headers from the [Readme template](https://github.com/microverseinc/readme-template) are available in your Readme 🙏